docs/how-to/how-to-set-sandbox-config-kata.md

```diff
@@ -57,6 +57,7 @@ There are several kinds of Kata configurations and they are listed below.
 | `io.katacontainers.config.hypervisor.enable_iothreads` | `boolean`| enable IO to be processed in a separate thread. Supported currently for virtio-`scsi` driver |
 | `io.katacontainers.config.hypervisor.enable_mem_prealloc` | `boolean` | the memory space used for `nvdimm` device by the hypervisor |
 | `io.katacontainers.config.hypervisor.enable_vhost_user_store` | `boolean` | enable vhost-user storage device (QEMU) |
+| `io.katacontainers.config.hypervisor.vhost_user_block_reconnect_timeout_sec` | `string`| the timeout for reconnecting SPDK (QEMU) 
```

Comment:
Is it reasonable to remove the `block` from the annotation/var name that let it be for all vhost user devices, though only block device is supported now?

Reply:
sounds like a good idea